### PR TITLE
Fix the namespacing of the gem

### PR DESCRIPTION
**Pull request by thisiscab**
There was inconsistent namespacing throughout the gem, this fixes that issue. 

Pretty much everything should now be namespaced under `Spree::Retail`.

### CODE REVIEW

**Comment by thisiscab** [comment]
@bryanmtl @braidn @DanielWright 

this is reaaaddyyy to rummbllleee!

**Comment by DanielWright** [comment]
Love it!

**Comment by braidn** [comment]
Will check it out, thanks for the refactor @cabouffard

File: spec/exporters/shopify/product_exporter_spec.rb [hunk 1]
```diff
@@ -1,65 +0,0 @@
-require 'spec_helper'
-
-# Given a Spree product
```
Comment (braidn) [inline spec/exporters/shopify/product_exporter_spec.rb:3]: Thank you! There is no point in committing commented out code.

File: spec/presenters/spree/retail/shopify/part_presenter_spec.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 require 'spec_helper'
 
-module Shopify
+module Spree::Retail::Shopify
```
Comment (braidn) [inline spec/presenters/spree/retail/shopify/part_presenter_spec.rb:3]: IMHO this is fine in specs. I know you asked this question about whether we keep this, the inline version or the tree version and honestly it's really whatever. If you want to enforce one way to write specs in the codebase that's chill but, I don't feel like it's 'gross' if we all write with our preferred style. Totally up to you though. 👍

**Review by braidn — COMMENTED** [review]
Very focused change here Cab. Nice.

**Comment by braidn** [comment]
@cabouffard this really should be golden besides the merge conflict.

**Pull request merged**